### DISCUSSION TITLE
Hybrids of Development Teams for Milestone 1

### DISCUSSION THREAD

**Discussion by lex939** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#63] [Informational]
Hello! If you are a team leader, please make sure to reply to this discussion with the two members of your team that will be working on a 50-50 split between documentation and development. These members will also be referred to as hybrids from now on. 

Each comment should contain the hybrids’ full names and usernames. Please make sure to also include the full name and username of the team leader. Thank you.

**Comment by yadrielsamir** [comment]
Admin Team
Team Leader: 
Yadriel Calderon Montalvo
@yadrielsamir 

Juan Diego Guadalupe Rosado
@Chosen-Juan1 

Jorge Colón Vélez
@JorgeC08

**Comment by Alejandro-DeJesus** [comment]
Trust System Team
Diego Velez @diego-velez-upr 
Jorge Rivera @JorgeRivera94

**Comment by shenied-maldonado** [comment]
User Account and Sessions 
Team leader: Shenied Maldonado (@shenied-maldonado)

Hybrids:
- John R Santiago (@johnsantiago9)
- Juan J Rivera (@JuanJRivera)

**Comment by lex939** [comment]
@yadrielsamir @shenied-maldonado @Alejandro-DeJesus Please edit your replies to also include full names of each hybrid. Please make sure to also include the full name and username of the team leader.

@Jjulianirizarry24 Please reply with the names and usernames of the hybrids on your team (Location Management Team). Make sure to also include the full name and username of the team leader.

**Reply by Jjulianirizarry24** [reply]
Location Management
Misael Mercado - @misaelmercado1
Alejandro Perez - @CaribbeanCool

**Comment by lex939** [comment]
The hybrids for Milestone 1 have been officially set.
